### PR TITLE
Clarify claim usage for signed token renewal.

### PR DESCRIPTION
**Pull request by PSUdaemon**
Augmented #29. @alficles can you comfirm this still meets your requirements?

### CODE REVIEW

**Comment by alficles** [comment]
Yeah, I think this might work. However, when reviewing, I noticed that we don't define a meaning for a value of 0 for cdnistt. I'm pretty sure 0 is supposed to mean "no signed token transport" and most people would probably guess that. But as it stands, it's not legal to provide a cdnistt claim with a value of 0. (Or at least, it's undefined and confusing.)

We should clarify that zero is a valid value. This will be of particular use to implementers that use JSON libraries that don't make it easy to conditionally omit members from objects when serializing them.

It might be valuable, in this case, to tighten up the language around what it means to "use" the cdnistt claim. I'd probably reword it to say "use a nonzero value in the cdnistt claim", or something like that. We don't want people writing validators that require a cdniets claim when a cdnistt claim of zero is present.

**Comment by PSUdaemon** [comment]
Currently we only show `1` as a valid value.

https://github.com/PSUdaemon/URISigningSpec/blob/master/draft-ietf-cdni-uri-signing.xml#L568

So really, 0 is not valid.

**Comment by alficles** [comment]
Yes, that's what I was observing. :) I was proposing we alter that, so that zero was defined to mean "no token transport" instead of just a lack of a claim. It will make implementing somewhat easier with some json libraries and very few, if any, implementations will have a problem with it.

And I think it makes the spec more consistent and readable, which will make implementations more likely to be correct. I think it's a small thing we can improve to make things as clear as we can make them and as easy to implement as is reasonable.
